controllers/firewall.go

```diff
@@ -46,8 +46,8 @@ func EnableMasquerade(ipmasq *networkv1alpha1.Masquerade) error {
 		return err
 	}
 
-	// After applying ebtables arp rules, it's better to send arp gratuitous request to make sure all overcloud
-	// nodes use proper mac for default gateway.
+	// After applying ebtables arp rules, it's better to send arp gratuitous request to make sure all Virtual Machines
+	// use proper mac for default gateway.
 	if err := arping.GratuitousArpOverIfaceByName(net.ParseIP(virtualIpaddress), ipmasq.Bridge); err != nil {
```

Comment:
I would prefer to add context to errors here and 2 more above. They all return err, but it will be hard to distinguish them once you get very generic error. e.g. 

```go
return fmt.Errorf("failed to send arp request. %v", err)
```

Reply:
great, I totally agree, done:)